### PR TITLE
Use UUID buckets.

### LINKED ISSUE #43: Bucket Name can't be more than 63 characters long

### LINKED ISSUE #45: Use UUID-based S3 Buckets

### PR DESCRIPTION
**Pull request by gozer**
Fixes #45 and fixes #43

### CODE REVIEW

File: nubis/cloudformation/main.json [hunk 1]
```diff
@@ -219,6 +243,12 @@
               "VpcInfo",
               "SharedServicesSecurityGroupId"
             ]
+          },
+          {
+            "Fn::GetAtt": [
```
Comment (limed) [inline nubis/cloudformation/main.json]: Being pedantic but this doesn't really have anything to do with uuid buckets

Comment (gozer) [inline nubis/cloudformation/main.json]: Pedantic, but also right! I'll split into another PR

**Comment by gozer** [comment]
Adjusted PR and made a new one #50

**Comment by limed** [comment]
r+

**Pull request merged**